.github/workflows/codeql-analysis.yaml

```
@@ -13,10 +13,10 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [main]
+    branches: [main, fix-codeql-scan-error]
```

```suggestion
    branches: [main]
```
Reminder: Revert before merge